packages/flutter/lib/src/services/asset_bundle.dart

```diff
@@ -237,9 +176,8 @@ AssetBundle _initRootBundle() {
           context: 'while initialising the root bundle'
         ));
       }
-      return true;
+      return false;
```

Comment:
I think we want an empty bundle for testing (or probably one that is configurable by the test---but we can start with an empty one).